framework/db/plugin_manager.py

```diff
@@ -43,9 +44,9 @@ def GetTestGroupsFromFile(self, file_path):  # This needs to be a list instead o
             TestGroups.append({'code': Code, 'priority': Priority, 'descrip': Descrip, 'hint': Hint, 'url': URL})
         return TestGroups
 
-    def LoadWebTestGroups(self, test_groups_file):
-        WebTestGroups = self.GetTestGroupsFromFile(test_groups_file)
-        for group in WebTestGroups:
+    def LoadTestGroups(self, test_groups_file, plugin_type):
```

Comment:
@DarKnight--  IMO you should rename `plugin_type` to `plugin_group` as from the OWTF structure plugin_types include passive, semi_passive, etc. and plugin_group includes web, net ,aux